.github/workflows/main-merge.yml

```diff
@@ -0,0 +1,18 @@
+# Merges any changes from release/prerelease to main (e.g. servicing changes)
+
+name: Flow main to release/dev18.0
+on:
+  schedule:
```

Comment:
Scheduled jobs run on the default branch and latest commit. See https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule